src/noir/common/log.cpp

```diff
@@ -0,0 +1,59 @@
+#include <noir/common/log.h>
+
+#include <fc/log/logger_config.hpp>
+
+#include <noir/tendermint/tendermint.h>
```

Comment:
```suggestion
#include <noir/common/log.h>
#include <noir/tendermint/tendermint.h>
#include <fc/log/logger_config.hpp>
```
Other includes follow the lexicographical sequence of (noir -> 3rd libs -> std libs).